app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new
+    @user.email = user_params[:email]
```

Comment:
I don't think we actually lose anything by leaving this explicit - it's easier to read and know at a glance exactly what parameters are being created